### DISCUSSION TITLE
[firebase_ml_vision] Adding the Object Detection Feature

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5555] [Feature request]
In the Firebase Ml Package there are the following Detectors : Face , Barcode , Image Labeling , Text Recognition , Landmark Detection. There is a new Detection model called Object Detection and you can find it in this link : [https://developers.google.com/ml-kit/vision/object-detection-and-tracking/](url) . It is part of ML Kit. So can you add it please as a new Feature.

**Comment by kroikie** [comment]
@Loopex2019

The issue at https://github.com/flutter/flutter/issues/34800 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by josefwilhelm** [comment]
Any news about when this feature will be available?
